server/gateway_test.go

```diff
@@ -3873,6 +3873,42 @@ func TestGatewayServiceImportWithQueue(t *testing.T) {
 	}
 }
 
+func ensureGWConnectTo(t *testing.T, s *Server, remoteGWName string, remoteGWServer *Server) {
```

Comment:
This is not specific to Windows, but they were often failing without that on Windows and sometimes on Travis. The idea is to "force" an outbound GW connection to a specific server.